spec/features/01_user_sign_in_spec.rb

```diff
@@ -0,0 +1,35 @@
+require 'rails_helper'
+
+feature "sees sign in options" do
+  scenario "see all sign in options" do
+    visit user_index_path
+    expect(page).to have_content('Sign In')
+    expect(page).to have_content('Sign Out')
```

Comment:
Do you want to see both "Sign In" and "Sign Out"? Wouldn't it be better to just see "Sign In" if no user is signed in and "Sign Out" if a user is signed in??